src/en/controllers-ha.md

```diff
@@ -2,11 +2,11 @@ Title: Juju High Availability
 TODO: Remove citation of HA logging bug when fixed (section: 'HA and logging')
 
 
-# High Availability
+# High availability
 
-Juju High Availability (HA) means that a Juju environment has 3 or more (up to
-7) state servers (bootstrap nodes) one of which is the *master*. Automatic
-failover occurs should the master lose connectivity.
+Juju high availability (HA) implies the presence of 3 or more (up to 7)
```

Comment:
the original wording was clumsy, but I don't think this is much of an improvement. It could do with more of an introduction. Maybe something along the lines of:

By default, Juju uses a single instance for a controller. However, when high availability (HA) is desired...

If you don't want to bother, put it in a TODO and I will take care of it later